tools/syz-crush/crush.go

```diff
@@ -52,7 +52,7 @@ func main() {
 		log.Fatal(err)
 	}
 	if *flagRestartTime == 0 {
-		*flagRestartTime *= cfg.Timeouts.VMRunningTime
+		*flagRestartTime = cfg.Timeouts.VMRunningTime
```

Comment:
will this require syzkaller config to have Timeouts? Is there a default timeout value?

Reply:
This is what sets the default timeout for VMs if you don't pass -restart_time flag for syz-crush.
With this fix you don't need to pass -restart_time. Without this fix the only way to set timeout is by passing -restart_time.

Reply:
I was wondering what the default value now was after this change. I figured `cfg` here was the syzkaller config file so thought the default value is now set by what is in the config file.

Reply:
This Timeouts object is not part of manager config, it's an internal detail:
https://github.com/google/syzkaller/blob/master/pkg/mgrconfig/config.go#L163-L165
https://github.com/google/syzkaller/blob/master/pkg/mgrconfig/load.go#L38

The default value is the same as what syz-manager will use as VM timeout.
If you are interested in the concrete value, I think it should be 1 hour for "adb" type.

Reply:
oh I see. Thank you.